packages/mobile/src/components/image/UserCoverImage.tsx

```diff
@@ -25,16 +25,28 @@ const interpolateImageTranslate = (animatedValue: Animated.Value) =>
   })
 
 type CoverImageUser = Nullable<
-  Pick<User, 'cover_photo_sizes' | 'cover_photo' | 'updatedCoverPhoto'>
+  Pick<
+    User,
+    | 'cover_photo_sizes'
+    | 'cover_photo_cids'
+    | 'cover_photo'
+    | 'updatedCoverPhoto'
+  >
 >
 
 export const useUserCoverImage = (user: CoverImageUser) => {
-  const cid = user ? user.cover_photo_sizes || user.cover_photo : null
+  let cid = null
+  if (user) {
+    cid = user.cover_photo_cids
+      ? user.cover_photo_cids[WidthSizes.SIZE_640]
```

Comment:
is there ever a chance that we dont have the cid for the specific size? ie should we check that cover_photo_cids exist and that the size specific one exists, otherwise fallback to cover_photo_sizes?

Reply:
good point - will add that in

Reply:
Sweet if that is the case one thing we can do is pass the cid map as a new arg instead of the "useRoute" param and we can check inside the function since we have access to the size?

Reply:
added in https://github.com/AudiusProject/audius-client/pull/3882/commits/d2f7677fb435ba48037ebd2fef41a1dc8c06b460!